src/Stryker.DataCollector/Stryker.DataCollector/CoverageCollector.cs

```diff
@@ -246,7 +246,11 @@ private void PublishCoverageData(DataCollectionContext dataCollectionContext)
                 _mutationCoveredOutsideTests);
 
             var coverData = testCoverageInfo.GetCoverageAsString();
-
+            if (string.IsNullOrEmpty(coverData))
```

Comment:
From the code perspective, I would be cautious relying on strings, they don't carry much information. Maybe `TestCoverageInfo` type can be extended instead?

Reply:
I am not sure what you have in mind here. We rely on VSTest extension infrastructure here, which only supports transmitting string properties, with the annoying limitation that you cannot transmit an empty string; if you try, the property is simply absent.
Stryker consider that an absent coverage property is a signal that the test was not reported to the coverage DataCollector; hence we need to test every mutant against it. Which is different from a lack of coverage, which implies the test does not cover any mutant.

So this test simply ensures we do not transmit an empty line

Reply:
That being said, it made me realize that it is not need as the DataCollector never returns an empty string. If a test covers no mutation, one gets a single semi-column. 
I had the impression fixed a problem I had with coverage capture, but I must have failed my tests somehow.